build/compile_descriptor

```diff
@@ -20,20 +20,26 @@ is_up_to_date () {
 
 set -e
 
+if [ -d "./ebin" ]; then
+    OUT="./ebin"
+else
+    OUT="$REBAR_DEPS_DIR/gpb/ebin"
+fi
+
```

Comment:
Also, thinking... After `rebar clean` (also after `make clean`), I have an empty ebin directory, would the if test need to be reversed (don't know at what stage rebar3 populates `_build/...`)